app/gui/src/project-view/util/ast/__tests__/abstract.test.ts

```diff
@@ -25,10 +26,19 @@ test('Raw block abstracts to Ast.BodyBlock', () => {
   expect(abstracted.root).toBeInstanceOf(Ast.BodyBlock)
 })
 
-//const disabledCases = [
-//  ' a',
-//  'a ',
-//]
+// Ideally anything would round-trip verbatim, but these odd inputs are currently known to be normalized by print/parse:
```

Comment:
This comment implies that this is undesirable, is that correct? Let's make that a little more clear if the intention is to fix those cases over time or not.